src/components/Cause/CauseBio/CauseBio.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react'
+import CauseBioDetail from './CauseBioDetail'
 
 const CauseBio = () => {
+  const content = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.'
+  const aboutTheCause = 'About the Cause'
```

Comment:
I think creating variables (aboutTheCause, aboutTheCharity) for the titles here might be an overkill as these titles are not really likely to be changed later.